src/xdocs/config_filters.xml

```diff
@@ -114,6 +114,14 @@
             Note that the suppression comment should be put before the violation. You can use more
             than one suppression comment each on separate line.
           </p>
+          <p>
+            Limitation: This filter may only be specified within the TreeWalker module
```

Comment:
Please change "Limitation" to "attention".
It is by design and intentional behavior, so it is not a limitation.